doc/user/flycheck-versus-flymake.rst

```diff
@@ -352,39 +250,25 @@ Error identifiers
 syntax checker provides these.  The identifiers appear in the error list and in
 error display, and can be copied independently, for instance for use in an
 inline suppression comment or to search the web for a particular kind of error.
+Some checkers can also provide more detailed error explanations based on these
+error identifiers.
 
```

Comment:
We could expand on this, maybe?

Reply:
Added a link to `flycheck-explain-error-at-point`.

Reply:
Well, if it is "some checkers" in flycheck, then I guess it's "backend-specific" in flymake. And thus, it's possible to do in Flymake too (see the manual for an example of a clickable&googleable error).

Reply:
@joaotavora No, not really: there's shared infrastructure in Flycheck to make it easier; it's just that not all backends implement it.